database/database.go

```diff
@@ -35,6 +36,15 @@ var (
 	// dbUsersCollection defines the name of the "users" collection within
 	// skynet's database.
 	dbUsersCollection = "users"
+	// dbSkylinksCollection defines the name of the "skylinks" collection within
+	// skynet's database.
+	dbSkylinksCollection = "skylinks"
+	// dbUploadsCollection defines the name of the "uploads" collection within
+	// skynet's database.
+	dbUploadsCollection = "uploads"
```

Comment:
`dbEvents` 🤔 ? in favour of having uploads and downloads?

Reply:
I don't see a use case in which we would need both in the same query. No need to strain those indexes if we don't need to. Unless there's a reason to prefer that?

Reply:
Some things would be slightly easier, like aggregating the amount of downloads/uploads per user, or finding latest user activitiy. Events would also be more generic, we could add more types of events over time.
Ignore it, we can always rename and refactor later.